build/emoji.js

```diff
@@ -0,0 +1,91 @@
+const axios = require('axios');
+const fs = require('fs');
+const path = require('path');
+
+const filePaths = {
+  emojiMarkdown: path.resolve(process.cwd(), 'docs', 'emoji.md'),
+  emojiJS: path.resolve(
+    process.cwd(),
+    'src',
+    'core',
+    'render',
+    'emojify-data.js'
+  ),
+};
+
+async function getEmojiData() {
+  const emojiDataURL = 'https://api.github.com/emojis';
+  const response = await axios.get(emojiDataURL);
+  const baseURL = Object.values(response.data)
+    .find(url => /unicode\//)
+    .split('unicode/')[0];
+  const data = { ...response.data };
+
+  // Remove base URL from emoji URLs
+  Object.entries(data).forEach(
+    ([key, value]) => (data[key] = value.replace(baseURL, ''))
+  );
```

Comment:
There are some URLs without `unicode/`, will it affect?

like https://github.githubassets.com/images/icons/emoji/electron.png?v8
https://github.githubassets.com/images/icons/emoji/octocat.png?v8

Reply:
When rendering image-based emoji, these URLs work fine. When rendering native emoji, the code falls back to rendering image-based emoji when it encounters one of the URLs.

Reply:
FYI, these are the emoji shorthand code that do not have unicode values associated with them in GitHub's API response:

* atom
* basecamp
* basecampy
* bowtie
* electron
* feelsgood
* finnadie
* goberserk
* godmode
* hurtrealbad
* neckbeard
* octocat
* rage1
* rage2
* rage3
* rage4
* shipit
* suspect
* trollface

Some of these (like `:atom:`) have [unicode values](https://emojipedia.org/atom-symbol/) associated with them but GitHub is not using them in their URLs for some reason. Others (like `:basecamp:`) are GitHub-specific, so the only way we can render them is using an image (as far as I know).